docs/source/user_guide/data_checks.ipynb

```diff
@@ -167,6 +319,91 @@
     "        warning_msg = \"Column '{}' has zero variance\"\n",
     "        return [DataCheckError(warning_msg.format(column), self.name) for column in X.columns if len(X[column].unique()) == 1]"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Data Checks\n",
```

Comment:
Let's change the title since we already introduced data checks at the top of this page.

I suggest we call this `## Defining Collections of Data Checks`